### DISCUSSION TITLE
2024/12/19/20241219_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#32] [General]
# 2024/12/19/20241219_updatelog/

今天拜读了巩高峰老师数年前的一篇文章 《婚姻这个话题，聊聊就聊聊》 ，结合最近一段时间的见闻和思考，想继续讨论一下婚姻和爱情的话题。

https://wz.anoms.top/2024/12/19/20241219_updatelog/

**Comment by THUDBT** [comment]
已阅

**Reply by cyclinbox** [reply]
踩踩，好久没看到樊总了欸
